doc/ROADMAP.rst.txt

```diff
@@ -147,9 +147,13 @@ are in good shape.
 
 linalg
 ``````
+``scipy.linalg`` is in good shape.  We have started requiring more recent
+LAPACK versions (minimum version increases from 3.2.1 to 3.4.0 in SciPy 1.2.0);
```

Comment:
Actually I think we were supporting 3.1.0 up until the jump to 3.4.0, though I don't thing anyone was testing that. Good idea adding this here though.

Reply:
Yes you're right. 3.2.1 was the minimum we could go up to if we weren't willing to drop Accellerate, but we had 3.1.0 support. Updated.